### ISSUE TITLE
Normalization is breaking FLS algo

### ISSUE THREAD

**Issue by tbreloff** [joshday/OnlineStats.jl#15]

**Comment by joshday** [comment]
I was just glancing at your code again and noticed your `normalize` functions are using variance instead of standard deviation.  I don't know if this will help, but it can't hurt.

After adding `Base.std(o::Var) = sqrt(var(o))` to `var.jl`, this should be 

```julia
normalize_y(o::OnlineFLS, y::Float64) = (y - mean(o.yvar)) / if0then1(std(o.yvar))
normalize_x(o::OnlineFLS, x::VecF) = (x - map(mean, o.xvars)) ./ map(x->if0then1(std(x)), o.xvars)
denormalize_y(o::OnlineFLS, y::Float64) = y * std(o.yvar) + mean(o.yvar)
denormalize_x(o::OnlineFLS, x::VecF) = x .* map(std, o.xvars) + map(mean, o.xvars)
```

**Comment by tbreloff** [comment]
Thanks I realized that as well... Base.std makes sense to include. The problem was actually that I was comparing a "normalized beta" to an "un-normalized beta", so whenever I normalized it seemed like the betas were no longer converging correctly. So (once I fixed variance to std dev) there wasn't actually a bug in the algo... I was just interpreting the results wrong. A few more methods/tests and I should be done with FLS.
